src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -156,6 +156,9 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						if (!value.isEmpty() && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H264)) {
 							media.setAvcLevel(getAvcLevel(value));
 						}
+						if (isNotBlank(value) && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H265)) {
```

Comment:
@Sami32 That's ok, the important thing is that ```media.getCodecV() != null``` comes before ```media.getCodecV().equals...```.

What I was trying to say is that it seemed strange to take the same information from MediaInfo, stored in ```value``` and store in two different places. But I see now that ```getAvcLevel()``` and ```getHvecLevel()``` produces different outcome, so I guess it's ok. Still, using one field to store this would mean you didn't have to expand the database etc to store this information twice.